test/fast/git-process-test.ts

```diff
@@ -195,7 +195,7 @@ describe('git-process', () => {
       try {
         await GitProcess.exec(['show', 'HEAD'], testRepoPath)
       } catch (e) {
-        error = e
+        error = e as Error
```

Comment:
In 4.8, `e` has type `unknown` under `--strict`. I added the one type assertion instead of turning that off